QModManager/Checks/VersionCheck.cs

```diff
@@ -66,16 +69,15 @@ internal static void Parse(string versionStr)
                 if (latestVersion > currentVersion)
                 {
                     Logger.Info($"Newer version found: {latestVersion.ToStringParsed()} (current version: {currentVersion.ToStringParsed()}");
-                    if (Patcher.ErrorModCount <= 0)
-                    {
-                        Dialog.Show(
-                            $"There is a newer version of QModManager available: {latestVersion.ToStringParsed()} (current version: {currentVersion.ToStringParsed()})",
-                            Dialog.Button.download, Dialog.Button.close, true);
-                    }
+                    result = latestVersion;
+                }
+                else if (latestVersion < currentVersion)
+                {
+                    Logger.Info($"Received latest version from GitHub. We're... ahead? Is this a development version, or did Prime forget to update the version again? 🙄");
```

Comment:
Try not to get too cheeky with the log messages.

A simple message like this will do
 `"Development build detected: Local version is ahead of server version."`